We.Sell.Bread.API/Services/CustomerService.cs

```diff
@@ -53,5 +61,11 @@ public async Task<bool> DeleteCustomerAsync(Guid id)
 
             return customer != false ? customer : false;
         }
+
+        public bool VerifyCustomerExists(string id)
+        {
+            return _customerRepository.GetCustomerById(id) == null ?
```

Comment:
Where is this method being used? could it be made private to the file? Could you have just added an if condition for the null check?

Reply:
The method is being used in the customer controller. 

It can definitely be a condition for null check seeing as we won't be reusing the method.